jax/_src/config.py

```diff
@@ -1528,6 +1528,14 @@ def _update_disable_jit_thread_local(val):
     default=True,
     help=('Enables using optimization-barrier op for lowering remat.'))
 
+enable_remat_opt_pass = bool_state(
+    name='jax_compiler_enable_remat_pass', 
+    default=False, 
```

Comment:
We certainly should check in this PR with the flag at its current default.

We can then explore, in a subsequent PR, flipping the default.